C/services/common/include/management_api.h

```diff
@@ -20,8 +20,9 @@
 #define PING			"/fledge/service/ping"
 #define SERVICE_SHUTDOWN	"/fledge/service/shutdown"
 #define CONFIG_CHANGE		"/fledge/change"
-#define CONFIG_CHILD_CREATE "/fledge/child_create"
-#define CONFIG_CHILD_DELETE "/fledge/child_delete"
+#define CONFIG_CHILD_CREATE     "/fledge/child_create"
+#define CONFIG_CHILD_DELETE     "/fledge/child_delete"
+#define SECURITY_CHANGE         "^/fledge/security$"
```

Comment:
@MarkRiddoch  Should this not be a well known URI, like change endpoint?

Reply:
@deepanshu-yadav the /fledge/security is a new endpoint for ACL management

Reply:
Ok taken that in my pull request.